.github/workflows/release_please.yml

```
@@ -54,3 +54,38 @@ jobs:
           skip_existing: true
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Get PR data
+        uses: actions/github-script@v7
+        id: get_pr_data
+        with:
+          script: |
```

```suggestion
          result-encoding: string
          script: |
```

Otherwise the return value is JSON encoded

---

Ah just noticed you return the whole PR, dismiss this